styles/style.css

```
@@ -95,7 +97,29 @@ p {
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+  button{
```

```suggestion
  button {
```
Leave a space between the CSS selector and the bracket. You can update this in a few other places too 🙂